src/OpenTelemetry/Metrics/Metric.cs

```diff
@@ -28,23 +28,31 @@ public sealed class Metric
     internal const int DefaultExponentialHistogramMaxScale = 20;
 
     internal static readonly double[] DefaultHistogramBounds = new double[] { 0, 5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500, 5000, 7500, 10000 };
-    internal static readonly double[] DefaultHistogramBoundsSeconds = new double[] { 0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 };
-    internal static readonly HashSet<(string, string)> DefaultHistogramBoundMappings = new()
+
+    // Short default histogram bounds. Based on the recommended semantic convention values for http.server.request.duration.
+    internal static readonly double[] DefaultHistogramBoundsShortSeconds = new double[] { 0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 };
+    internal static readonly HashSet<(string, string)> DefaultHistogramBoundShortMappings = new()
     {
         ("Microsoft.AspNetCore.Hosting", "http.server.request.duration"),
-        ("Microsoft.AspNetCore.Http.Connections", "signalr.server.connection.duration"),
         ("Microsoft.AspNetCore.RateLimiting", "aspnetcore.rate_limiting.request.time_in_queue"),
         ("Microsoft.AspNetCore.RateLimiting", "aspnetcore.rate_limiting.request_lease.duration"),
-        ("Microsoft.AspNetCore.Server.Kestrel", "kestrel.connection.duration"),
         ("Microsoft.AspNetCore.Server.Kestrel", "kestrel.tls_handshake.duration"),
         ("OpenTelemetry.Instrumentation.AspNetCore", "http.server.request.duration"),
         ("OpenTelemetry.Instrumentation.Http", "http.client.request.duration"),
-        ("System.Net.Http", "http.client.connection.duration"),
         ("System.Net.Http", "http.client.request.duration"),
         ("System.Net.Http", "http.client.request.time_in_queue"),
         ("System.Net.NameResolution", "dns.lookups.duration"),
     };
 
+    // Long default histogram bounds. Not based on a standard. May change in the future.
+    internal static readonly double[] DefaultHistogramBoundsLongSeconds = new double[] { 0, 0.01, 0.02, 0.05, 0.1, 0.2, 0.5, 1, 2, 5, 10, 30, 60, 120, 300 };
```

Comment:
Let's remove the bucket with the value `0`?

- https://github.com/open-telemetry/semantic-conventions/pull/283#discussion_r1378202685
- https://github.com/dotnet/docs/pull/37811/files#r1378201879

Reply:
The other buckets bounds have a zero value. I think this PR should focus on one thing, and if removing zero makes sense then it can be a new PR.

Reply:
FYI - we will fix the other buckets as well for 1.7.0 release
https://github.com/open-telemetry/opentelemetry-dotnet/issues/4908

Reply:
I don't mind if its one PR or two, but we should fix now while we can, and before it gets adopted - although I don't think its a breaking change.